administrator/language/en-GB/com_guidedtours.ini

```
@@ -0,0 +1,76 @@
+COM_GUIDEDTOURS_FIELD_VALUE_INTERACTIVE_STEP="Interactive"
+COM_GUIDEDTOURS_FIELD_VALUE_NEXT_STEP="Next"
+COM_GUIDEDTOURS_FIELD_VALUE_REDIRECT="Redirect"
+COM_GUIDEDTOURS_FILTER_SEARCH_DESC="Search in id, title and description. Prefix with ID: to search for tour ID or DESCRIPTION: to search only description text."
+COM_GUIDEDTOURS_GUIDEDTOUR_STEPS_PAGE_TITLE="Guided Tour - List of Steps"
+COM_GUIDEDTOURS_GUIDEDTOURS_LABEL_DESCRIPTION="Description"
+COM_GUIDEDTOURS_GUIDEDTOURS_PAGE_TITLE="Guided Tours - List of Tours"
+COM_GUIDEDTOURS_GUIDEDTOURS_TABLE_CAPTION="List of Tours"
+COM_GUIDEDTOURS_INTERACTIVE_STEP_DESC="Click on the type of interactive step which you want to select"
+COM_GUIDEDTOURS_N_ITEMS_DELETED_1="Tours deleted."
+COM_GUIDEDTOURS_N_ITEMS_DELETED="%s tours deleted."
+COM_GUIDEDTOURS_N_ITEMS_PUBLISHED_1="Tours enabled."
+COM_GUIDEDTOURS_N_ITEMS_PUBLISHED="%s tours enabled."
+COM_GUIDEDTOURS_N_ITEMS_TRASHED_1="Tours trashed."
+COM_GUIDEDTOURS_N_ITEMS_TRASHED="%s tours trashed."
+COM_GUIDEDTOURS_N_ITEMS_UNPUBLISHED_1="Tours disabled."
+COM_GUIDEDTOURS_N_ITEMS_UNPUBLISHED="%s tours disabled."
+COM_GUIDEDTOURS_N_ITEMS_ARCHIVED_1="Tours archived."
+COM_GUIDEDTOURS_N_ITEMS_ARCHIVED="%s Tours archived."
+COM_GUIDEDTOURS_ORDER_DESCRIPTION_ASC="Description ascending"
+COM_GUIDEDTOURS_ORDER_DESCRIPTION_DESC="Description descending"
+COM_GUIDEDTOURS_ORDER_TYPE_ASC="Type ascending"
+COM_GUIDEDTOURS_ORDER_TYPE_DESC="Type descending"
+COM_GUIDEDTOURS_POSITION_DESC="Add the Position where you want to display your step. e.g. Bottom, Top, Right, Left, Centered"
+COM_GUIDEDTOURS_POSITION_LABEL="Position"
+COM_GUIDEDTOURS_RULES_TAB="Permissions"
+COM_GUIDEDTOURS_STATUS="Status"
+COM_GUIDEDTOURS_STEP_ASC="Step ascending"
+COM_GUIDEDTOURS_STEP_DESC="Step descending"
+COM_GUIDEDTOURS_STEP_FILTER_SEARCH_DESC="Search in id, title and description. Prefix with ID: to search for step ID or DESCRIPTION: to search only description text."
+COM_GUIDEDTOURS_STEP_ID="ID"
+COM_GUIDEDTOURS_STEP_TITLE="Title"
+COM_GUIDEDTOURS_STEP_TYPE="Type"
+COM_GUIDEDTOURS_STEP_TYPE_NEXT_STEP="Next"
+COM_GUIDEDTOURS_STEP_TYPE_REDIRECT="Redirect"
+COM_GUIDEDTOURS_STEP_TYPE_INTERACTIVE_STEP="Interactive"
+COM_GUIDEDTOURS_STEPS="Steps"
+COM_GUIDEDTOURS_STEPS_EMPTYSTATE_CONTENT="A tour can only be functional if steps are created."
+COM_GUIDEDTOURS_STEPS_EMPTYSTATE_TITLE="No step for this tour has been created yet."
+COM_GUIDEDTOURS_STEPS_EMPTYSTATE_BUTTON_ADD="Add your first step"
+COM_GUIDEDTOURS_STEPS_LIST="Guided Tour: %s"
+COM_GUIDEDTOURS_TARGET_DESC="Add the target item to attach your step element to. Options: .classname, #id or leave blank for a centered step."
+COM_GUIDEDTOURS_TARGET_LABEL="Target"
+COM_GUIDEDTOURS_TITLE="Guided Tours"
+COM_GUIDEDTOURS_TOUR_ID="ID"
+COM_GUIDEDTOURS_TOUR_TITLE="Title"
+COM_GUIDEDTOURS_TOURS_LIST="Guided Tours"
+COM_GUIDEDTOURS_TYPE_LABEL="Type"
+COM_GUIDEDTOURS_TYPE_DESC="Click on the type which you want to select for the tour"
```

```suggestion
COM_GUIDEDTOURS_TYPE_DESC="Select the type which you want to select for the tour."
```

We don't use the word click - touch screen, accessibility